ui/components/app.jsx

```diff
@@ -7,7 +7,7 @@ function App(props) {
   return (
     <div>
       <Disclaimer />
-      <Navbar />
+      <Navbar router={props.router} />
```

Comment:
We might be able to add the router as a `contextType` of the `Search` component. Then we wouldn't need to thread it through each of the intervening components. Looks like we have an example in the `Autocompleter`